integration/generator/resources/ec2_linux_test_matrix.json

```diff
@@ -96,5 +96,14 @@
     "caCertPath": "/etc/ssl/certs/ca-bundle.crt",
     "arc": "amd64",
     "binaryName": "amazon-cloudwatch-agent.rpm"
+  },
+  {
```

Comment:
No blocker from me: Do we want to test on every different version of the same OS when we have an upper bound and lower bound? For example: oracle linux 8-2 is no different than ol 8-1 (not any breaking changes being introduced so far); therefore, only ol8-1 and ol8-6 would be enough for CWAgent (open suggestion here)

Reply:
I think it should work that way as long as we totally understand what's changing between versions and any side effects.

Reply:
Agree 👍 . Will suggest this in parking lot and see how its go from there.